src/Http/Transport/Bridge/PsrHttpClient/PsrHttpClientTransport.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * BigBlueButton open source conferencing system - https://www.bigbluebutton.org/.
```

Comment:
Since the code is not from BBB, I think we can use the recommended lgpl header.

Reply:
I hope I used the recommended header you meant. On gnu.org, I only found a definitive header for the GPL, not LGPL.